### DISCUSSION TITLE
"hide/show" desktop? (in essence)

### DISCUSSION THREAD

**Discussion by oddyeus** [i3/i3#5689] [Q&A]
I'm using two monitors with i3wm.
Is it possible to use a single keybind (for example bindsym + `) to show the next unused/new workspaces on both monitors, then the same keybind to return to the previous workspaces on both monitors?

Essentially what I'm looking for is a way to "hide/show" the desktop with a single keybind, or even two individual keybinds if thats not possible.

**Comment by orestisfl** [comment]
You can do that with the IPC interface. See https://github.com/i3/i3/discussions/5536 as an inspiration and you can work your way from there
